### PR TITLE
refactor: move inline runtime imports to dedicated modules

### PR DESCRIPTION
**Pull request by jansenk**
**TL;DR -** move inline runtime imports to dedicated modules

JIRA: [AU-574](https://openedx.atlassian.net/browse/AU-574)

**Developer Checklist**

- [ ] Reviewed the [release process](https://github.com/edx/edx-ora2/blob/master/.github/release_process.md)
- [ ] Translations and JS/SASS compiled
- [ ] Bumped version number in [setup.py](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/setup.py#L39) and [package.json](https://github.com/edx/edx-ora2/blob/a62e81a9b0d89223476967ec3c27f3557a850735/package.json#L3)

**Reviewer Checklist**

Collectively, these should be completed by reviewers of this PR:

- [ ] I've done a visual code review

FYI: @openedx/content-aurora

### CODE REVIEW

**Comment by jansenk** [comment]
I'm most likely just gonna ignore the coverage failure. I touched data.py but I take no responsibility for the sins committed there

**Review by nsprenkle — APPROVED** [review]
Much better :shipit:
